src/symbolic/ellipsoidal_transitions.jl

```diff
@@ -562,7 +559,9 @@ function transition_backward(
 
     @variable(model, t)
     u_q = [L[i, j] for j in 1:nx for i in 1:j]
+
     @constraint(model, vcat(t, 1, u_q) in MOI.LogDetConeTriangle(nx))
+    # @constraint(model, vcat(t, u_q) in MOI.RootDetConeTriangle(nx))
```

Comment:
Remove comment

Reply:
Or say explicitely in a comment that we can also put a RootDetConeTriangle, and explain how it changes \lambda